oauthenticator/oauth2.py

```diff
@@ -239,6 +239,13 @@ class OAuthLogoutHandler(LogoutHandler):
     async def handle_logout(self):
         self.clear_cookie(STATE_COOKIE_NAME)
 
+    async def render_logout_page(self):
+        if self.authenticator.logout_redirect_url:
+            self.redirect(self.authenticator.logout_redirect_url)
+            return
+
+        super().render_logout_page()
+
```

Comment:
To ensure proper extension of possibly async method, we must *either*:

- use `async def` and  await the super result, or
- use `def` and directly return the super result so it can be awaited

```suggestion
   def render_logout_page(self):
        if self.authenticator.logout_redirect_url:
            self.redirect(self.authenticator.logout_redirect_url)
            return

        return super().render_logout_page()

```

Reply:
One last thing, then 👍 to merge.